fury/io.py

```diff
@@ -223,9 +230,12 @@ def save_image(arr, filename, compression_quality=75,
     if use_pillow:
         arr = np.flipud(arr)
         im = Image.fromarray(arr)
-        im.save(filename, quality=compression_quality)
+        im.save(filename, quality=compression_quality, dpi=dpi)
         return
 
+    if dpi != (0, 0):
+        warnings.warn(UserWarning('DPI value is ignored while saving images via vtk.'))
```

Comment:
I saw the original PR comment, and @Nibba2018's question related to specifying the DPIs to the `VTK` writer. I know this was not going to be easy, but I think it should be eventually implemented.

As the current PR is already an improvement, I would merge it upon testing success but would keep the issue open; once the PR merge, I can add a word saying that using the DPI value for the `VTK` writer is still to be done.

Reply:
@Nibba2018 It is some time since I have not worked with `VTK` so I probably do not have all the details with me. I did have a quick look before submitting the issue, and when reviewing your PR again, and did not find any immediate documentation. If you feel that you are willing to implement specifying DPIs for a `VTK` image, you may want to consider:
- Investigating how they accomplish this in other `VTK`-based projects (e.g. `PyVista`. `Slicer 3D` ?)
- Asking in the `VTK` discourse forum

Hope this helps.